src/components/picker.js

```diff
@@ -68,7 +68,7 @@ export default class Picker extends React.Component {
     this.categories = CATEGORIES;
   }
 
-  componentWillReceiveProps(props) {
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
FYI, this should be easily converted to `getDerivedStateFromProps`

Reply:
Ah. Sweet!